Assets/Plugins/Source/Editor/Windows/CreatePackageWindow.cs

```diff
@@ -311,7 +353,7 @@ private async void StartCreatePackageAsync(UPMUtility.PackageType type)
             {
                 _operationInProgress = false;
                 _progressText = "";
-                _progress = 0f;
+                _actualProgress = 0f;
```

Comment:
Should we tell _progressUpdateThread to stop here?

Reply:
Nope - what's really nice about the way this is done is that the `_progressUpdateThread` is created using a `CancellationToken`, and all we need to do is cancel the token, and it will be handled for us. 

For the sake of simplicity, you can use the same token for multiple threads. In our case - we have one thread that's copying files, and one thread that's pausing the UI briefly in order to make the progress bar update smoothly - both circumstances can use the same `CancellationToken` so we can cancel them simultaneously.